src/platform/Infineon/CYW30739/EventFlags.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    Copyright (c) 2019 Nest Labs, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *    Provides an implementation of the event flags for the platform.
```

Comment:
this `@file` doc comment seems like a waste of vertical space: of coure a file named `EventFlags` inside `src/platform/....` provides an implementation of events flags for a platform. It is also very vague `the event flags` (what are they) and `the platform` (does not even bother to state what platform ... since it is obvious in the path anyway.

I would prefer if we would not have such non-info `@file` entries in our source code.